README.md

```diff
@@ -98,14 +103,92 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       left => right
     ```
 
+###  **December 6 - LCM**
+- **Problem**
+  - Return the least common multiple of two or more numbers.
+  - Use the `greatest common divisor (GCD)` formula and the fact that `lcm(x,y) = x * y / gcd(x,y)` to determine the least common multiple.
+  - **Hint:** The GCD formula might use recursion.
+- **Sample Run**
+  ```bash
+  lcm(12, 7) # 84
+  lcm([1, 3, 4], 5) # 60
+  ```
+
+###  **December 7 - Are they approximately equal?**
+- **Problem**
+  - Checks if two numbers are approximately equal to each other.
```

Comment:
Explain what you mean by "approximately equal". I'm assuming that you're rounding the numbers here. But better to be clear.